inst/stan/marginal_model/functions.stan

```diff
@@ -0,0 +1,10 @@
+real primarycensored_wrapper_lpmf(data int d, real mu, real sigma, data real pwindow) {
+  int dist_id = 1; // lognormal
+  array[2] real params;
+  params[1] = mu;
```

Comment:
This needs to be generalised to any `dpars`

Reply:
Approach here would be to:

1. use regex for the `real mu, real sigma` as with the latent model
2. look at the family and extract out the name and match to some table to get to the `dist_id`
  * Is there a way to get this table from `primarycensored` so we don't have to keep it up to date here
  * Is there a family agnostic version of `primarycensored_lpmf`? i.e. one which isn't tied to particular `dist_id` or analytic solutions
  
@seabbs thoughts here?